src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ParameterAssignmentCheck.java

```diff
@@ -190,14 +190,12 @@ private void visitIncDec(DetailAST ast) {
      * @param ast ident to check.
      */
     private void checkIdent(DetailAST ast) {
-        if (!parameterNames.isEmpty()) {
```

Comment:
This is just an early check before calling `parameterNames.contains`. If the list is empty, `contains` won't return true anyways, so it all amounts to the same.